package.json

```diff
@@ -56,25 +58,27 @@
     "querystring-es3": "^0.2.1",
     "ramda": "^0.26.1",
     "react": "16.8.3",
+    "react-i18next": "^9.0.10",
     "react-native": "^0.59.10",
     "react-native-action-sheet": "^2.2.0",
     "react-native-browser": "^0.6.2",
-    "react-native-cell-components": "^0.7.5",
+    "react-native-cell-components": "^0.7.8",
     "react-native-collapsible": "^0.8.0",
-    "react-native-crypto": "^2.0.2",
+    "react-native-crypto": "^2.2.0",
     "react-native-dropdownalert": "^3.11.0",
     "react-native-elements": "^0.19.0",
     "react-native-exception-handler": "^1.1.0",
-    "react-native-gesture-handler": "^1.3.0",
-    "react-native-google-signin": "^2.0.0",
-    "react-native-keychain": "^3.0.0",
+    "react-native-gesture-handler": "^1.5.2",
+    "react-native-google-signin": "^2.1.1",
+    "react-native-keychain": "^3.1.3",
     "react-native-level-fs": "^3.0.0",
     "react-native-loading-spinner-overlay": "^0.5.2",
+    "react-native-localize": "^1.3.1",
     "react-native-navbar": "^1.7.1",
-    "react-native-os": "^1.2.5",
+    "react-native-os": "^1.2.6",
     "react-native-progress-wheel": "^1.0.4",
     "react-native-randombytes": "^3.5.3",
-    "react-native-reanimated": "^1.2.0",
+    "react-native-reanimated": "^1.4.0",
```

Comment:
Does this compile? I'm pretty sure `react-native-reanimated` 1.4.0 doesnt work with RN < 0.60
https://github.com/software-mansion/react-native-reanimated/issues/488#issuecomment-559462197

Reply:
I'll take a look at it. Maybe it was because I was running `npm update`.

Reply:
I set it back to `1.2.0`.